apps/obs-web/src/views/DebrieferAssessmentEditor.vue

```diff
@@ -0,0 +1,348 @@
+<template>
+    <div class="q-pa-md q-gutter-md">
+
+        <q-table
+          title="SECTIONS"
+          :data="sections"
+          :columns="sectionsColumns"
+          row-key="_id"
+          binary-state-sort
+          dense
+        >
+          <template v-slot:body="props">
+            <q-tr :props="props">
+              <q-td key="sections" :props="props">
+                {{ getArrayString(props.row.sections) }}
+                <q-popup-edit v-model="props.row.sections" buttons persistent @save="save(props.row)">
+                  <div v-for="section of props.row.sections" :key="props.row.sections.indexOf(section)">
+                    <q-input v-model="props.row.sections[props.row.sections.indexOf(section)]" dense>
+                      <template v-slot:append>
+                        <q-btn icon="clear" flat @click="props.row.sections.splice(props.row.sections.indexOf(section), 1)"></q-btn>
+                      </template>
+                    </q-input>
+                  </div>
+                    <q-btn flat style="float: right; margin-top: 10px" @click="props.row.sections.push('')">add section</q-btn>
+                </q-popup-edit>
+              </q-td>
+              <q-td key="isWcgop" :props="props">
+                {{ props.row.isWcgop }}
+                <q-popup-edit v-model="props.row.isWcgop" buttons persistent @save="save(props.row)">
+                    <q-select v-model="props.row.isWcgop" :options="[true, false]" dense></q-select>
+                </q-popup-edit>
+              </q-td>
+              <q-td key="isAshop" :props="props">
+                {{ props.row.isAshop }}
+                <q-popup-edit v-model="props.row.isAshop" buttons persistent @save="save(props.row)">
+                    <q-select v-model="props.row.isAshop" :options="[true, false]" dense></q-select>
+                </q-popup-edit>
+              </q-td>
+            </q-tr>
+          </template>
+        </q-table>
+
+        <q-table
+          title="QUESTIONS"
+          :data="questions"
+          :columns="questionColumns"
+          row-key="_id"
+          binary-state-sort
+          dense
+          :pagination.sync="questionsPagination"
+        >
+        <template v-slot:top :props="props">
+          <div style="width: 100%; font-size: 1.5em; margin-top: 5px">
+            <b style="color: #007EC6">QUESTIONS</b>
+            <span style="color: #007EC6; font-size: .6em; vertical-align: middle; margin-left: 10px; font-style: italic">default sort order: program/section/order</span>
+            <q-btn style="float: right" flat color="primary" @click="addQuestion">new question</q-btn>
```

Comment:
Didn't notice this button at first, could probably remove flat so it shows up more prominently. Also is there a way to delete a record if the new question button is added by accident?

Reply:
was on the fence how prominent these buttons should be - will remove 'flat'